driver/examples/configs/baroclinic_c12_from_serialbox.yaml

```diff
@@ -8,6 +8,9 @@ initialization_type: serialbox
 initialization_config:
   path: /test_data/c12_6ranks_baroclinic_dycore_microphysics
   serialized_grid: True
+performance_config:
+  timer: true
```

Comment:
is this not always true for any performance run?  would it help if we just had a way to run in performance mode and have the performance config default to true/true?

Reply:
This is a good point. Changing it to just one config.